### ISSUE TITLE
Train/fit a keras model directly on losses instead using inputs and labels

### ISSUE THREAD

**Issue by kk-55** [keras-team/keras#14076]
Can I also train/fit a keras model directly on losses instead using `model.fit(x=inputs, y=labels)`?
I wanna clip the error to be between some interval (e.g. (tf.float32.min, 0)).

**Comment by klausk55** [comment]
Okay the length of my example error intervall should be much too large I think (leading to exploding error gradients in my opinion). Right?

**Comment by keshav5196** [comment]
You can use callbacks such as ModelCheckpoint and earlystopping to monitor the loss and stop training if the loss increases.

**Comment by klausk55** [comment]
But if I understand you right, your suggestion let me monitor the loss but not use it as input to directly train/fit a model. Is there a way to give a keras model the loss to do a "backward pass" updating the weights?

**Comment by keshav5196** [comment]
Hi,

I found a page which explains how to use our custom loss during backpropagation.  Here's the link:

https://heartbeat.fritz.ai/how-to-create-a-custom-loss-function-in-keras-637bd312e9ab

**Comment by jvishnuvardhan** [comment]
@kk-55 This is a stale issue. Sorry for late response. 

Is this still an issue for you? Can you describe the issue with more details? Are you trying to train your model with a custom loss function?

Can you please share a simple standalone code to reproduce the issue? Also, check your code with the recent TF versions (`2.6` and `tf-nightly`) and let us know whether the issue persists with recent TF versions. Thanks!

**Comment by kk-55** [comment]
@jvishnuvardhan to be honest, I can't remember the concret context of this issue. Therefore I just close it and say thank you nevertheless!
